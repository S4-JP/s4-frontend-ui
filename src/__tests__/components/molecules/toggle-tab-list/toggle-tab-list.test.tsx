import React from 'react'
import { fireEvent, render, screen } from '@testing-library/react'
import { ToggleTabList } from '../../../../index'

const Dummy = () => <span>dummy</span>

describe('トグルタブリストのレンダリング確認', () => {
  test('propsに設定した値でレンダリングされる。', () => {
    render(
      <ToggleTabList>
        <Dummy />
      </ToggleTabList>
    )

    expect(screen.getByTestId('molecules-ttl-wrap')).toMatchSnapshot()
  })
})
