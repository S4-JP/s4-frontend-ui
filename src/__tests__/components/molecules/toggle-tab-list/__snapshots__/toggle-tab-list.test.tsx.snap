// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`トグルタブリストのレンダリング確認 propsに設定した値でレンダリングされる。 1`] = `
<div
  class="sc-iWajrY gKIHjj"
  data-testid="molecules-ttl-wrap"
>
  <span>
    dummy
  </span>
</div>
`;
